contracts/staking/ValidatorManager.sol

```diff
@@ -321,13 +306,20 @@ abstract contract ValidatorManager is
         emit ValidationPeriodEnded(validationID, validator.status);
     }
 
+    /**
+     * @notice Returns the validator's weight. This weight is not guaranteed to be known by the P-Chain
+     */
+    function getWeight(bytes32 validationID) external view returns (uint64) {
+        return _getValidator(validationID).weight;
+    }
```

Comment:
Should we require that this returns a non-zero value? Or leave it up to the caller?

Reply:
If we don't want to require it here, we should leave a comment that a return value of 0 means the validation ID doesn't exist.

Reply:
Updated the comment to clarify the behavior for unregistered validation IDs